app/league/config_utils.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""Helper utility for managing application settings."""
+import json
+from os.path import exists, getsize, join
+
+from flask import current_app
+
+
+def update_config_file(new_config):
```

Comment:
No rush. I would look [here](https://github.com/massgo/league/blob/master/app/league/dashboard/models.py) for examples of SQLAlchemy entities. I think it'll probably be enough to have an "AdminConfig" or similar entity that is essentially a key-value store, along with a method that gets fired using the `before_first_request_funcs` stuff you can see [here](https://github.com/massgo/league/blob/master/app/league/app.py#L88). Otherwise, SQLAlchemy docs are your friend. After you do this, ou will need to create a db migration using `flask db migrate`, edit and commit the resulting migration.

This is probably the most conceptually tricky part of this whole app, so please feel free to hit me up on Slack with questions. Once you have a handle on this stuff you will be able to do basically anything on the backend :)